server/lib/payments.js

```diff
@@ -676,7 +676,7 @@ const sendOrderConfirmedEmail = async (order, transaction) => {
     // normal order
     const data = {
       order: order.info,
-      transaction: transaction ? transaction.info : null,
+      transaction: transaction ? transaction.info : new Date(),
```

Comment:
Isn't transaction  in data an object?
```suggestion
      transaction: transaction ? transaction.info : { createdAt: new Date() },
```